diff --git a/node_modules/genkit-cli/lib/commands/start.js b/node_modules/genkit-cli/lib/commands/start.js
index 92d86a4..a96f131 100644
--- a/node_modules/genkit-cli/lib/commands/start.js
+++ b/node_modules/genkit-cli/lib/commands/start.js
@@ -53,7 +53,10 @@
         shutdown();
         process.exit(0);
     });
-    const (await Promise.resolve().then(() => __importStar(require('node:http')))).createServer(requestListener).listen(options.port);
+    const http = (await Promise.resolve().then(() => __importStar(require('node:http'))));
+    const server = http.createServer(requestListener);
+    await new Promise((resolve) => server.listen(options.port, resolve));
+
     console.log(`Your Genkit flows are available at http://localhost:${options.port}`);
     if (options.watch) {
         console.log('Watching for changes in the current directory...');

